docs/source/usage.rst

```diff
@@ -492,6 +492,43 @@ Note how the port changes to 82, and how the users lists are combined.
       file: log.txt
     <BLANKLINE>
 
+By default, ``merge()`` is replacing the target list with the source list.
+Use ``list_merge_mode`` to control the merge behavior for lists. 
+You can access the regarding enum with ``from omegaconf import ListMergeMode``.
+Currently there are three different merge modes:
+* ``OVERRIDE``: content from newer list gets taken (default)
+* ``EXTEND``: lists get extended
+* ``EXTEND_IGNORE_DUPLICATES``: only new (unique) elements get extended
+
+**example2.yaml** file:
+
+.. include:: example2.yaml
+   :code: yaml
+
+**example4.yaml** file:
+
+.. include:: example4.yaml
+   :code: yaml
+
+If you load them and merge them with ``list_merge_mode=ListMergeMode.EXTEND_IGNORE_DUPLICATES`` you will get this:
+
+.. doctest::
+
+    >>> from omegaconf import OmegaConf, ListMergeMode
+    >>>
+    >>> cfg_example2 = OmegaConf.load('source/example2.yaml')
+    >>> cfg_example4 = OmegaConf.load('source/example4.yaml')
+    >>> 
+    >>> conf = OmegaConf.merge(cfg_example2, cfg_example4, list_merge_mode=ListMergeMode.EXTEND_IGNORE_DUPLICATES)
+    >>> print(OmegaConf.to_yaml(conf))
```

Comment:
Some lines here could be too long and cause the docs to have a horizontal scroll bar.
You can build the docs locally to check that. I suspect you will need to format the code a bit with some newlines.

Reply:
Fixed in the latest iteration.